test/tls.uts

```diff
@@ -984,10 +1064,32 @@ t.len = 0xefff
 str(t) == b'\xde\xad\xbe\xef\xff\x01'
 
 
+= Building packets - TLS record with bad data
+a = TLS(b'\x00\x03\x03\x00\x03data')
+assert a.haslayer(Raw)
+
+
+= Building packets - _CipherSuitesField with no cipher
+from scapy.layers.tls.handshake import _CipherSuitesField
+a = _CipherSuitesField("test", None, {})
+assert a.i2repr(None, None) == "None"
+assert isinstance(a.randval(), RandBin)
+assert hash(a) == 8890308
```

Comment:
I didn't pay attention to the new tests to tls.uts in #637 and I reorganized them a little bit.
@gpotter2 are you sure the hash assertion is always True?